app/src/test/java/uk/gov/onelogin/MainActivityViewModelTest.kt

```diff
@@ -57,13 +41,9 @@ class MainActivityViewModelTest {
     )
 
     private val viewModel = MainActivityViewModel(
-        mockContext,
-        mockLoginSession,
-        mockCredChecker,
         mockBioPrefHandler,
         mockTokenRepository,
-        mockAutoInitialiseSecureStore,
-        mockVerifyIdToken
+        mockAutoInitialiseSecureStore
     )
 
     @BeforeEach
```

Comment:
A LiveData `observeForever()` without the corresponding `removeObserver()`. Pairs of functions should have both available

Reply:
fixed